### PR TITLE
Integrate Aurora Design System

### PR DESCRIPTION
**Pull request by caleybrock**
# Description

This PR removes [React-Bootstrap CSS](https://react-bootstrap.github.io/getting-started/introduction/) and replaces it with CSS from the [Aurora Design System](https://design.gccollab.ca/overview/introduction). This design system is a lightweight CSS-only version of gov look and feel standards (similar to WET) that is being used by other government departments. This means we can implement our own React components instead of having to use HTML only, but still follow government CSS standards. There are also HTML examples in the design system that we can make into React components, which was really helpful in this PR.

There are 4 files added to a `/css` folder in the repository that are downloaded from Aurora and can be updated/replaced at any time.

I went through and removed many react-bootstrap component and replaced it with HTML and CSS classes from the Aurora Design System. The good news is that the classnames seem to be very similar, so if we decide to use both, they should work together nicely, for the most part.

Next steps is to use `App.css` as an override to customize our colours to PSC look and feel standards (what Joey has been developing) and make the header/nav look more realistic. 

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation update
- [x] Code cleanliness or refactor

## Screenshot

This is the Status Page which had the most changes. Other pages look almost the same with the exception of some font changes.
image

# Testing

Manual steps to test this functionality: Click through entire application to see design changes.

Any concerns that need to be addressed in product usability testing: I still need to implement the header but I decided to leave this for a separate PR.

Screenshot of unit tests run passing:
screen shot 2019-02-13 at 4 52 25 pm

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new compiler warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have researched WCAG2.1 standards and met them in this PR - given the HTML structure of tables, I believe my changes to be accessible.

We can use any CSS we want and still be accessible so Auora should be good to go. Their examples are WCAG2.1 which is very helpful.
I can't find a section of Bootstrap that says it conforms to WCAG, but it does seem to[ tab complete](https://react-bootstrap.netlify.com/components/tabs/#tabs-with-dropdown), which is promising. 
I found this link helpful too: https://coreui.io/docs/getting-started/accessibility/ 

Because of this, it seems promising to be able to use both of these (Aurora CSS + React Bootstrap components) together.

### CODE REVIEW

File: frontend/src/components/status/StatusCheck.jsx [hunk 1]
```diff
@@ -41,23 +23,23 @@ class StatusCheck extends React.PureComponent {
...
-        <div style={styles.icon}>
+      <tr>
+        <td>{description}</td>
+        <td>
           {isPassing && (
             <div style={styles.passing}>
-              <Glyphicon style={styles.glyphicon} glyph="ok-circle" />
-              <Label>{LOCALIZE.commons.passStatus}</Label>
+              <i className="fas fa-check-circle" />
+              <span>{LOCALIZE.commons.passStatus}</span>
```
Comment (fnormand01) [inline frontend/src/components/status/StatusCheck.jsx]: Can we add a space between _**< span >**_ and _**{LOCALIZE.commons.passStatus}**_, to have a little space between the check mark and the Pass word like this:

image

File: frontend/src/components/status/StatusCheck.jsx [hunk 2]
```diff
@@ -41,23 +23,23 @@ class StatusCheck extends React.PureComponent {
...
             </div>
           )}
           {!isPassing && (
             <div style={styles.failing}>
-              <Glyphicon style={styles.glyphicon} glyph="remove-circle" />
-              <Label>{LOCALIZE.commons.failStatus}</Label>
+              <i className="fas fa-times-circle" />
+              <span>{LOCALIZE.commons.failStatus}</span>
```
Comment (fnormand01) [inline frontend/src/components/status/StatusCheck.jsx]: Same comment as my previous one, but for the fail status:

Can we add a space between _**< span >**_ and _**{LOCALIZE.commons.passStatus}**_, to have a little space between the check mark and the Fail word?

File: frontend/src/components/status/SystemCheck.jsx [hunk 3]
```diff
@@ -47,25 +24,25 @@ class SystemCheck extends React.PureComponent {
...
+        <td>{description}</td>
+        <td>
           {isPassing && (
             <div style={styles.passing}>
-              <Glyphicon style={styles.glyphicon} glyph="ok-circle" />
-              <Label>{LOCALIZE.commons.passStatus}</Label>
-              <p style={styles.currentSpecs}>{currentSettingsDetails}</p>
+              <i style={styles.glyphicon} className="fas fa-check-circle" />
+              <span>{LOCALIZE.commons.passStatus}</span>
+              <span style={styles.currentSpecs}>{currentSettingsDetails}</span>
```
Comment (fnormand01) [inline frontend/src/components/status/SystemCheck.jsx]: Can we add a space between _**< span >**_ and _**{LOCALIZE.commons.passStatus}**_ and between _**< span style={styles.currentSpecs} >**_ and _**{currentSettingsDetails}**_, to have a little spaces between the check mark, the Pass word and the settings details like this:

image

File: frontend/src/components/status/SystemCheck.jsx [hunk 4]
```diff
@@ -47,25 +24,25 @@ class SystemCheck extends React.PureComponent {
...
             </div>
           )}
           {!isPassing && (
             <div style={styles.failing}>
-              <Glyphicon style={styles.glyphicon} glyph="remove-circle" />
-              <Label>{LOCALIZE.commons.failStatus}</Label>
-              <p style={styles.currentSpecs}>{currentSettingsDetails}</p>
+              <i style={styles.glyphicon} className="fas fa-times-circle" />
+              <span>{LOCALIZE.commons.failStatus}</span>
+              <span style={styles.currentSpecs}>{currentSettingsDetails}</span>
```
Comment (fnormand01) [inline frontend/src/components/status/SystemCheck.jsx]: Same comment as my previous one, but for the fail status:

Can we add a space between _**< span >**_ and _**{LOCALIZE.commons.failStatus}**_ and between _**< span style={styles.currentSpecs} >**_ and _**{currentSettingsDetails}**_, to have a little spaces between the check mark, the Pass word and the settings details?

**Comment by caleybrock** [comment]
Updated - feel free to take a look again.

**Comment by fnormand01** [comment]
Now it looks better. Thanks for these updates.

**Pull request merged**